modules/logging-loki-storage-aws.adoc

```diff
@@ -25,3 +25,31 @@ $ oc create secret generic logging-loki-aws \
   --from-literal=access_key_secret="<aws_access_key_secret>" \
   --from-literal=region="<aws_region_of_your_bucket>"
 ----
+
+== AWS storage for STS enabled clusters
+
+You can create a secret manually by running the following command:
```

Comment:
Can we rename it as `You can create the Loki object storage secret manually by running the following command` simply because it might cause confusion with the secret that is created by the CCO.

Reply:
I changed the order they were presented in to try to avoid the same thing, so yes absolutely I'll change that now. :)